### PR TITLE
Add location api

### PR DESCRIPTION
**Pull request by arturs105**
Ref #2169
Ref #1960

I was kinda split between using `string`s for country name and code vs `ICountry`, but i went with strings to keep it closer to the json that backend returns.

Squash whenever.

### CODE REVIEW

File: Toggl.Ultrawave/Models/Location.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Newtonsoft.Json;
+using Toggl.Multivac.Models;
+
+namespace Toggl.Ultrawave.Models
+{
+    internal sealed partial class Location : ILocation
+    {
+        [JsonProperty("city")]
```
Comment (simonrozsival) [inline Toggl.Ultrawave/Models/Location.cs]: This shouldn't be needed.

File: Toggl.Ultrawave/Models/Location.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        public string City { get; set; }
+
+        [JsonProperty("state")]
```
Comment (simonrozsival) [inline Toggl.Ultrawave/Models/Location.cs]: This shouldn't be needed.

File: Toggl.Ultrawave/Models/Location.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        public string State { get; set; }
+
+        [JsonProperty("country_name")]
+        public string Country { get; set; }
```
Comment (simonrozsival) [inline Toggl.Ultrawave/Models/Location.cs]: This property should be called `CountryName` so the property is not needed.

**Comment by amulware** [comment]
`docs/endpoints.md` is lonely :troll:

**Pull request merged**